### PR TITLE
OCPBUGS-35726: fix cert/key file reload

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Comment by simonpasquier** [comment]
/hold

hmm the server doesn't seem to pick up the new certificate... Doing more tests.

File: cmd/plugin-backend.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ var (
 	staticPathArg       = flag.String("static-path", "", "static files path to serve frontend (default: './web/dist')")
 	configPathArg       = flag.String("config-path", "", "config files path (default: './config')")
 	pluginConfigArg     = flag.String("plugin-config-path", "", "plugin yaml configuration")
-	logLevelArg         = flag.String("log-level", "error", "verbosity of logs\noptions: ['panic', 'fatal', 'error', 'warn', 'info', 'debug', 'trace']\n'trace' level will log all incoming requests\n(default 'error')")
+	logLevelArg         = flag.String("log-level", logrus.InfoLevel.String(), "verbosity of logs\noptions: ['panic', 'fatal', 'error', 'warn', 'info', 'debug', 'trace']\n'trace' level will log all incoming requests\n(default 'error')")
```
Comment (simonpasquier) [inline cmd/plugin-backend.go:21]: @PeterYurkovich I've changed back the default log level to info so I could see the logs when the certificate gets updated. It also brings back logs such as "listening for https on :9443..." which are useful in general from my POV.

The [logs](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_monitoring-plugin/264/pull-ci-openshift-monitoring-plugin-main-e2e-aws-ovn/1861781336927244288/artifacts/e2e-aws-ovn/gather-extra/artifacts/pods/openshift-monitoring_monitoring-plugin-78874749d9-nslpq_monitoring-plugin.log) from the CI runs don't show the repeated access logs as mentioned in https://github.com/openshift/monitoring-plugin/pull/186. And I've checked also with a test cluster. Feel free to object if you think that we should stick to `error` by default.

**Comment by simonpasquier** [comment]
@PeterYurkovich @zhuje this is good for review now.

**Review by Elno5 — APPROVED** [review]

**Review by Elno5 — APPROVED** [review]

**Pull request merged**